storyscript/compiler/lowering/Faketree.py

```diff
@@ -68,13 +68,24 @@ def mark_line(self, node, line):
             else:
                 self.mark_line(child, line)
 
+    def set_line(self, node, line):
+        """
+        Sets the given `line` on the first token in node's subtree or
+        on the node itself if the subtree is void of token's
```

Comment:
1) tokens not token's
2) void doesn't fit here
3) end with dot
4) its subtree

```suggestion
        on the node itself if its subtree doesn't contain tokens.
```

Reply:
will update :+1: